src/components/List/Tasks.js

```diff
@@ -0,0 +1,154 @@
+import { Typography } from "@material-ui/core";
+import { MdDelete } from "react-icons/md";
+import IconButton from "@material-ui/core/IconButton";
+import { updateListmodul } from "../../firebase/updateListmodul";
+import { deleteListItem } from "../../firebase/deleteListItem";
+import { getUserData } from "../../firebase/getUserData";
+import { AiOutlineInfoCircle } from "react-icons/ai";
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    padding: theme.spacing(2.5),
+  },
+  root: {
+    width: "100%",
+    maxWidth: 360,
+  },
+  info: {
+    display: "flex",
+    marginTop: theme.spacing(2),
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  infoIcon: {
+    fontSize: "1.5rem",
+    marginRight: theme.spacing(1),
+  },
+  center: {
+    textAlign: "center",
+    margin: theme.spacing(3, 0),
+  },
+  title: {
+    margin: theme.spacing(1, 0),
+  },
+}));
+
+export default function Tasks({ tasks, update, category }) {
+  const classes = useStyles();
+
+  const sortByDateAndCateogry = tasks
+    .filter((task) => task.list === category)
+    .sort(function (a, b) {
+      const dateA = new Date(a.date),
+        dateB = new Date(b.date);
+      return dateA - dateB;
+    });
+
+  const renderTaskByCategory = (status) => {
+    const isDone = status === "done" ? true : false;
+    const taskBycategory = sortByDateAndCateogry
+      .filter((task) => task.done === isDone)
+      .map((task, idx) => (
+        <List dense className={classes.root} key={idx}>
+          <Task task={task} update={update} />
+        </List>
+      ));
+
+    if (taskBycategory.length === 0 && status === "open") {
+      return (
+        <Typography
+          variant="subtitle1"
+          color="textSecondary"
+          className={`${classes.info} ${classes.center}`}
+        >
+          <AiOutlineInfoCircle className={classes.infoIcon} />
+          Du hast keine offenen Aufgaben
+        </Typography>
+      );
+    }
+    return taskBycategory;
+  };
+
+  return (
+    <div className={classes.wrapper}>
+      <Typography variant="h2" className={classes.title}>
+        Aufgaben
+      </Typography>
+      {renderTaskByCategory("open")}
+      {renderTaskByCategory("done").length !== 0 && (
+        <>
+          <Typography variant="h2" className={classes.title}>
+            Erledigt
+          </Typography>
+          {renderTaskByCategory("done")}
+        </>
+      )}
+    </div>
+  );
+}
+
+function Task({ task, update }) {
+  const [profileImage, setProfileImage] = useState();
+
+  const handleChecked = (e) => {
+    updateListmodul(task.docId, !task.done);
+    update();
+  };
+
+  const handleDelete = (e) => {
+    deleteListItem(task.docId);
+    update();
+  };
+
+  useEffect(() => {
+    const loadUserData = async () => {
+      const userData = await getUserData(task.assignedTo);
+      setProfileImage(userData.profileImage);
+    };
+    loadUserData();
+  }, [task.assignedTo]);
+
+  return (
+    <ListItem button>
```

Comment:
```suggestion
    <ListItem style={{ paddingLeft: "0" }}>
```
Mit dem `button` bekommt man die Card Action bzw. diesen Klick-Effekt was in diesem Kontext keinen Sinn ergibt. Das Padding kommt weg, damit alles in einer Flucht ist 😃